generators/server/templates/src/main/java/package/service/dto/_UserDTO.java

```diff
@@ -109,6 +109,10 @@ public String getLogin() {
         return login;
     }
 
+    public void setLogin(String login) {
```

Comment:
The data from UserDTO class is used to send back to the front-end.
The login of the user is used by default to be displayed (in the table without pager).
So the login in UserDTO need to be set from the User (mapstruct mechanism, if there is not setter, the value is not 'copied')